bigframes/series.py

```diff
@@ -1557,6 +1558,10 @@ def __array_ufunc__(
     def str(self) -> strings.StringMethods:
         return strings.StringMethods(self._block)
 
+    @property
+    def plot(self):
```

Comment:
Docstring, please.

Also, please add PlotAccessor to https://github.com/googleapis/python-bigquery-dataframes/blob/main/docs/reference/bigframes.pandas/series.rst

Reply:
Thanks! Added `PlotAccessor` to the `series.rst` and `frames.rst`. 
As for the docstring for `def plot`, it is defined at third_party/bigframes_vendored/pandas/core/series.py